CMake/fletch-tarballs.cmake

```diff
@@ -154,6 +154,23 @@ set(Eigen_md5 "6a578dba42d1c578d531ab5b6fa3f741")
 set(Eigen_dlname "eigen-${Eigen_version}.tar.gz")
 list(APPEND fletch_external_sources Eigen)
 
+#OpenBLAS
+if(NOT WIN32)
+  #set(OpenBLAS_version "0.2.15")
+  set(OpenBLAS_version "0.2.20")
+  set(OpenBLAS_url "https://github.com/xianyi/OpenBLAS/archive/v${OpenBLAS_version}.tar.gz")
+
+  if (OpenBLAS_version VERSION_EQUAL 0.2.20)
+    set(OpenBLAS_md5 "48637eb29f5b492b91459175dcc574b1")
+  elseif (OpenBLAS_version VERSION_EQUAL 0.2.15)
+    set(OpenBLAS_md5 "b1190f3d3471685f17cfd1ec1d252ac9")
```

Comment:
Do we really need both packages? OpenBLAS is more of an incidental package, not used directly by out algorithms.

Reply:
probably not.